packages/workflows/src/handlers/inventory/attach-inventory-items.ts

```diff
@@ -7,26 +7,22 @@ export async function attachInventoryItems({
   data,
 }: WorkflowArguments<{
   inventoryItems: {
-    variant: ProductTypes.ProductVariantDTO
+    tag: string
     inventoryItem: InventoryItemDTO
   }[]
 }>) {
-  const { manager } = context
-
-  const productVariantInventoryService = container
-    .resolve("productVariantInventoryService")
-    .withTransaction(manager)
```

Comment:
@carlos-r-l-rodrigues - is it okay to remove the transaction managers and rely on the workflow orchestrator instead?

Reply:
No it is not when it comes to medusa services, because they are singleton and calling withTransaction allow us to get a new instance. Otherwise the manager of another request might be used here and we get back to the already released error. Though, the manager it not a transaction manager or not necessarily and is used to just get this new instance

Reply:
Got it will revert 👍